### PR TITLE
Tytynono66 patch 1

### PR DESCRIPTION
**Pull request by tytynono66**
is that correct now?

### CODE REVIEW

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 1]
```diff
@@ -402,7 +402,7 @@
         </trans-unit>
         <trans-unit id="ModernNavigationToolbaNewTextDocument.Text" translate="yes" xml:space="preserve">
           <source>Text Document</source>
-          <target state="new">Text Document</target>
+          <target state="new">Texte Document</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.fr-FR.xlf:405]: Make sure to change the state to `translated`.

**Comment by tytynono66** [comment]
?? how ? where change the state to translated

Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

De : Yair Aichenbaum<mailto:notifications@github.com>
Envoyé le :vendredi 24 avril 2020 21:40
À : duke7553/files-uwp<mailto:files-uwp@noreply.github.com>
Cc : tytynono66<mailto:tytynono@msn.com>; Author<mailto:author@noreply.github.com>
Objet :Re: [duke7553/files-uwp] Tytynono66 patch 1 (#682)


@yaichenbaum commented on this pull request.


In Files/MultilingualResources/Files.fr-FR.xlf<https://github.com/duke7553/files-uwp/pull/682#discussion_r414818357>:

> @@ -402,7 +402,7 @@

         </trans-unit>

         <trans-unit id="ModernNavigationToolbaNewTextDocument.Text" translate="yes" xml:space="preserve">

           <source>Text Document</source>

-          <target state="new">Text Document</target>

+          <target state="new">Texte Document</target>

Make sure to change the state to translated.

**Comment by yaichenbaum** [comment]
Just open the file in a text editor and change it there.

**Comment by xPoppyx** [comment]
translated

**Comment by tytynono66** [comment]
it's done, is everything ok?


tytynono

De : Richard<mailto:notifications@github.com>
Envoyé le :vendredi 24 avril 2020 21:48
À : duke7553/files-uwp<mailto:files-uwp@noreply.github.com>
Cc : tytynono66<mailto:tytynono@msn.com>; Author<mailto:author@noreply.github.com>
Objet :Re: [duke7553/files-uwp] Tytynono66 patch 1 (#682)


[translated]<https://user-images.githubusercontent.com/61259627/80251034-e9669880-8642-11ea-8f33-4305282cd920.gif>

**Comment by yaichenbaum** [comment]
@tytynono66 Can you make the requested changes?

**Pull request closed without merging**